packages/SwingSet/src/kernel/state/storageHelper.js

```diff
@@ -26,7 +26,10 @@ export function* enumeratePrefixedKeys(kvStore, prefix, exclusiveEnd) {
   }
 }
 
-// NOTE: awkward naming: the thing that returns a stream of keys is named
+/**
+ * @param {KVStore} kvStore
+ * @param {string} prefix
+ */ // NOTE: awkward naming: the thing that returns a stream of keys is named
```

Comment:
How does this change get deployed?
Do we have plans to upgrade the kernel bundle?

Reply:
I don't know. Since it's just JSDoc does it matter?

Reply:
looks like this isn't the 1st such change in any case: 9aa5868